cube_browser/bld.bat

```diff
@@ -0,0 +1 @@
+python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
This magic line requires setuptools (not a core python library). Either cube_browser depends on setuptools, or this line can simply be `python setup.py install`.